sktime/transformations/series/tests/test_differencer.py

```diff
@@ -42,8 +42,11 @@ def test_differencer_produces_expected_results(na_handling):
 
 @pytest.mark.parametrize("y", test_cases)
 @pytest.mark.parametrize("lags", lags_to_test)
-def test_differencer_same_series(y, lags):
+@pytest.mark.parametrize("index_type", ["int", "datetime"])
```

Comment:
Not sure if this is better. As an alternative, we could add the time series with resetted index also to the `test_cases` list.

Reply:
that would double the number of test cases, and fixture paramterization logic in the module root - I think there is already too much logic in the module root and not in `pytest.fixture`-s ...

Reply:
if one wants to be really clean, it should be a parametric pytest fixture